src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -539,7 +560,17 @@ def __init__(self,
     self._afl_input = None
     self._afl_output = None
 
-    self.strategies = FuzzingStrategies()
+    logs.log(
```

Comment:
let's make this logging more general and move this to generate_new_testcase_mutations. Also, right now this will log even if we aren't doing a mutations run, which doesn't seem too useful.